ETC.tsv

```diff
@@ -16088,41 +16088,41 @@ ETC_20151022_016089	We should at least block that...
 ETC_20151022_016090	Then our villagers...
 ETC_20151022_016091	I saw it with my own eyes while running away. What the demons were doing.
 ETC_20151022_016092	I will help too!
-ETC_20151022_016093	Avoid the surveillance sphere that circulates around the surveillance crystal{nl}and eliminate the magic field which manages the surveillance crystal across!
+ETC_20151022_016093	Avoid the surveillance sphere that circulates around the surveillance crystal,{nl}and remove the magic circle that manages the surveillance crystal across!
 ETC_20151022_016094	Surveillance Crystal
-ETC_20151022_016095	Surveillance Crystal Magic Field
-ETC_20151022_016096	The demons that are carrying a gigantic axe
-ETC_20151022_016097	Move Fast. So lazy!
+ETC_20151022_016095	Surveillance Crystal Magic Circle
+ETC_20151022_016096	Demons that are carrying a gigantic axe
+ETC_20151022_016097	Move fast. So lazy!
 ETC_20151022_016098	Help!!
 ETC_20151022_016099	Ferrets! Ferrets appeared near the village!
-ETC_20151022_016100	Damn... worthless human!!
-ETC_20151022_016101	Run away! Soon, Zaura will come!
-ETC_20151022_016102	The observer disappeared after taking the incomplete Kruvina!{nl}Defeat the awakened Zaura!
-ETC_20151022_016103	The Awakened Demon Lord Zaura
+ETC_20151022_016100	You worthless human!!
+ETC_20151022_016101	Run away! Zaura will come soon!
+ETC_20151022_016102	The Beholder disappeared after taking the incomplete Kruvina!{nl}Defeat the awakened Zaura!
+ETC_20151022_016103	Awakened Demon Lord Zaura
 ETC_20151022_016104	The demon with an gigantic axe disappeared with the girl!{nl}Defeat the Ferrets that are rushing in!
 ETC_20151022_016105	Damn! Hey, Help me!
 ETC_20151022_016106	Kwaaak!!!
-ETC_20151022_016107	Damn! They... again!
+ETC_20151022_016107	What! They... again!
```

Comment:
if you put what ..isn't it supposed to have '?' in there? since it have already become a question. Personally, I prefer the original one . Also there's a typo in 16119